### DISCUSSION TITLE
Separate Rest API or not

### DISCUSSION THREAD

**Discussion by 6nagi9** [vercel/next.js#47828] [Help]
### Summary

We plan to build a NextJS website and a Flutter app over a common database with similar business logic. Should we develop a separate Rest API using Express and NodeJS or create endpoints in the NextJS website codebase itself? What is the best practice?

**Comment by larssn** [comment]
I'd build an express server myself. Seems cleaner, and not dependent on NextJS. Would also make it easy to migrate it to something like Firebase Hosting.
